specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-10-05-privatepreview/examples/ServerGroupCreate.json

```diff
@@ -17,6 +17,7 @@
         "enableZfs": false,
         "postgresqlVersion": "12",
         "citusVersion": "9.5",
+        "availabilityZone": "1",
```

Comment:
Let's add the standbyAvailabilityZone too.

Reply:
Done